src/coord/src/coord.rs

```diff
@@ -413,65 +409,63 @@ impl Coordinator {
         self.send_builtin_table_updates(builtin_table_updates).await;
 
         // Announce primary and foreign key relationships.
-        if self.logging_granularity.is_some() {
```

Comment:
Removing this `if` seems like it will likely have broken `materialized --logging-granularity=off`?

Reply:
That option (now called `--introspection-frequency`) still worked here and disabled the logging tables, and I'm not sure what dataflow did when it received these updates? It doesn't panic for some reason I didn't trace down. Anyhow I've replaced this option with a boolean so we can disable this branch, which seems good enough for now.